scripts/hail_batch/variant_selection/hgdp_1kg_tob_wgs_variant_selection.py

```diff
@@ -68,16 +68,15 @@ def query(output):  # pylint: disable=too-many-locals
     pruned_variant_table = hl.ld_prune(
         hgdp1kg_tobwgs_joined.GT, r2=0.1, bp_window_size=500000
     )
+    pruned_variant_table = pruned_variant_table.cache()
+    nrows = pruned_variant_table.count_rows()
+    print(f'pruned_variant_table.count_rows() = {nrows}')
     hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.filter_rows(
         hl.is_defined(pruned_variant_table[hgdp1kg_tobwgs_joined.row_key])
     )
     mt_path = f'{output}/tob_wgs_hgdp_1kg_filtered_variants.mt'
-    tmp_path = (
-        'gs://cpg-tob-wgs-main-tmp/tob_wgs_hgdp_1kg_variant_selection/'
-        'tob_wgs_hgdp_1kg_filtered_variants.mt.tmp'
-    )
-    hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.checkpoint(tmp_path)
-    hl.read_matrix_table(tmp_path, _n_partitions=1000).write(mt_path)
+    hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.repartition(1000, shuffle=False)
```

Comment:
That means we can use secondary (i.e. preemptible) workers now, right?

Reply:
Yes, with `shuffle=False`. This is going off the same way we did it before, which used 20 preemptible workers

Reply:
Are you sure? `main.py` still specifies non-preemptible workers, as far as I can tell: https://github.com/populationgenomics/ancestry/blob/main/scripts/hail_batch/variant_selection/main.py#L23